src/autoComplete/AutoComplete.story.mdx

```diff
@@ -0,0 +1,178 @@
+import { Meta, Preview, Props, Story } from '@storybook/addon-docs/blocks'
+
+import { Button, ButtonStatus } from '../button'
+import { SearchIcon } from '../icon'
+import { BaseSection } from '../layout/section/baseSection'
+import { AutoCompleteExample } from './AutoCompleteExample'
+import { AutoComplete } from './index'
+
+<Meta title="AutoComplete" />
```

Comment:
```suggestion
<Meta title="Forms|AutoComplete" />
```
AutoComplete is a form element lets bring it to this category to improve discoverability